src/app/work-item/work-item-list/work-item-list-entry/work-item-list-entry.component.scss

```diff
@@ -63,6 +63,15 @@ span{
   }
 }
 
+.form-group {
+  margin: 15px auto;
```

Comment:
form-group is a common class and will be used at various places. Please use custom classes or more hierarchy for custom styles